Apps/Sandcastle/gallery/CameraLimiter Tutorial.html

```diff
@@ -0,0 +1,211 @@
+        outline : true,
+        outlineColor : Cesium.Color.YELLOW,
+        slicePartitions : 10,
+        stackPartitions : 10
+    }
+});
+
+
+
+
+// Used for AxisAlignedBoundingBox Example
+var centerForAxisAligned = Cesium.Cartesian3.fromDegrees(-100.0, 45.0, 45.0);
+
+var dimensionsForAxisAligned = new Cesium.Cartesian3(300000.0, 300000.0, 500000.0);
+var halfDimensionsAxisAligned = Cesium.Cartesian3.multiplyByScalar(dimensionsForAxisAligned, 0.5, new Cesium.Cartesian3());
+
+var minForAxisAligned = new Cesium.Cartesian3(-halfDimensionsAxisAligned.x, -halfDimensionsAxisAligned.y, -halfDimensionsAxisAligned.z);
+var maxForAxisAligned = new Cesium.Cartesian3(halfDimensionsAxisAligned.x, halfDimensionsAxisAligned.y, halfDimensionsAxisAligned.z);
+
+// normalizing by dividing by magnitude and offsetting along direction vector so outside of earth surface
+var offsetForAxisAligned = 10000.0;
+var directionForAxisAligned = Cesium.Cartesian3.multiplyByScalar(centerForAxisAligned,
+    offsetForAxisAligned / Cesium.Cartesian3.magnitude(centerForAxisAligned),
+    new Cesium.Cartesian3());
+Cesium.Cartesian3.add(centerForAxisAligned, directionForAxisAligned, centerForAxisAligned);
+
+// The Axis Aligned Box
+var axisAligned = new Cesium.AxisAlignedBoundingBox(minForAxisAligned, maxForAxisAligned, centerForAxisAligned);
+
+// View the Axis Aligned Bounding Box
+var outlineAxisBox = viewer.entities.add({
+    name : 'Axis Aligned Box Outline',
+    position: centerForAxisAligned,
+    box : {
+        dimensions : new Cesium.Cartesian3(dimensionsForAxisAligned.x, dimensionsForAxisAligned.y, dimensionsForAxisAligned.z),
+        fill : false,
+        outline : true,
+        outlineColor : Cesium.Color.RED
+    }
+});
+
+
+
+
+// Used for OrientedBoundingBox Example
+var centerForOriented = Cesium.Cartesian3.fromDegrees(-90.0, 45.0, 45.0);
+var dimensionsForOriented = new Cesium.Cartesian3(300000.0, 300000.0, 500000.0);
+var halfDimensionsOriented = Cesium.Cartesian3.multiplyByScalar(dimensionsForOriented, 0.5, new Cesium.Cartesian3());
+
+var minForOriented = new Cesium.Cartesian3(-halfDimensionsOriented.x, -halfDimensionsOriented.x, -halfDimensionsOriented.x);
+var maxForOriented = new Cesium.Cartesian3(halfDimensionsOriented.x, halfDimensionsOriented.y, halfDimensionsOriented.z);
+
+// normalizing by dividing by magnitude and offsetting along direction vector so outside of earth surface
+var offsetForOriented = 10000.0;
+var directionForOriented = Cesium.Cartesian3.multiplyByScalar( centerForOriented,
+    offsetForOriented / Cesium.Cartesian3.magnitude(centerForOriented),
+    new Cesium.Cartesian3());
+Cesium.Cartesian3.add(centerForOriented, directionForOriented, centerForOriented);
+
+// The Oriented Box
+var oriented = new Cesium.AxisAlignedBoundingBox(minForOriented, maxForOriented, centerForOriented);
+
+// View the Oriented Bounding Box - NEEED A BETTERRR WWAAAYYY TO DRAWWW (bbox is correct - visual isnt oriented the way it is.)
```

Comment:
What the issue here?

Reply:
I dont think the oriented bounding box is actually rotated per the orientation of the box - i think the visual box that i have is actually just axis aligned (so it's not necessarily a good representation for this case but works well for the axis aligned version)